validator/src/cli/thread_args.rs

```diff
@@ -213,6 +218,17 @@ impl ThreadArg for IpEchoServerThreadsArg {
     }
 }
 
+struct RayonGlobalThreadsArg;
+impl ThreadArg for RayonGlobalThreadsArg {
+    const NAME: &'static str = "rayon_global_threads";
+    const LONG_NAME: &'static str = "rayon-global-threads";
+    const HELP: &'static str = "Number of threads to use for the global rayon thread pool";
+
+    fn default() -> usize {
+        get_max_thread_count()
+    }
```

Comment:
Is 1 (the default) okay for the min here or should we bump it?

Reply:
I think it's ok - if someone specifies a bad value for a hidden argument that is on them IMO

On a somewhat related note, I think trying to get most operations (if not everything) out of the global pool is a good goal; the global pool is easy to use but given that anyone other service could be lighting it up with work, not great to have that uncertainty.